### DISCUSSION TITLE
SFRKC30.AT2 fw 3.3 loading saved games

### DISCUSSION THREAD

**Discussion by Prince-Eugen** [keirf/flashfloppy#657] [General]
Hi, I have an issue with gotek board SFRKC30.AT2 with flash floppy fw 3.3 stable version (bought flashed from store). 
Usb with .adf files is fat32 formatted. Beside those .adf files there are also two files in root folder of usb (autoboot.hfe and hxcsdfe.cfg).
Configuration is amiga 500 with 512 chip RAM + 1.5MB fast RAM + kick 1.3.
Loading .adf files works properly, but the problem is with game save discs.  Amiga successfully creates save game disc (Dune 2, Sensible Soccer...) with empty disc created via WinUAE but it shows Disc error when loading saved games.
Referred issue does not appear in WinUAE software with mentioned games and their .adfs so saving and loading via game discs works properly.
JC jumper on the board is not marked but that position is open as it should be for Amiga.
How to resolve this or is there any additional configuration ? 
Thx.

**Comment by keirf** [comment]
Is the save disk image an 880k ADF in each case?

**Reply by Prince-Eugen** [reply]
yes, i have updated to 3.22 latest fw, but still disk error appears on Amiga 500

**Reply by keirf** [reply]
That doesn't answer the question, and 3.22 is not the latest v3 series firmware.

**Comment by Prince-Eugen** [comment]
i have use standard 3.5 dd image for all cases.
maybe i could upload saved games from amiga and from winuae, just to compare these 2 adfs to see what causes error in reading ?

**Reply by Prince-Eugen** [reply]
it is  fw 3.32  at the moment,  it was a typo

**Reply by keirf** [reply]
Sure you can zip and upload them to this ticket.

**Comment by Prince-Eugen** [comment]
saved disk wont work from amiga to winuae and vice versa.

here are the saved images from sensible soccer 96-97 as example.
[winuae sensible 96-97 working save disk.zip](https://github.com/keirf/flashfloppy/files/8881811/winuae.sensible.96-97.working.save.disk.zip)
[amiga sensible 96-97 nonworking save disk.zip](https://github.com/keirf/flashfloppy/files/8881820/amiga.sensible.96-97.nonworking.save.disk.zip)
![IMG_20220610_212303](https://user-images.githubusercontent.com/68650447/173138165-d478055e-3efa-4edb-b788-7e1db12db798.jpg)

**Reply by keirf** [reply]
Do you mean a save game disk from winuae works in winuae but you can't load that same game on gotek?

**Reply by Prince-Eugen** [reply]
yes on winuae everything works perfect, but on gotek disk error when trying to load saved status,
i think this maybe is issue with protection bits---rwed or some kind of protection of gotek.

**Reply by Prince-Eugen** [reply]
i repeat this is not only issue with this game, some games wont even save status on amiga

**Reply by keirf** [reply]
It does make it a different issue though if a game known correctly saved on winuae won't even load on amiga+gotek. That makes it not (only) a gotrk save issue.

**Comment by Prince-Eugen** [comment]
just to clear, position of jumpers by default
![IMG_20220610_222319](https://user-images.githubusercontent.com/68650447/173145651-9c78342c-eb14-4c1d-ae96-547cbe8774ff.jpg)

**Comment by keirf** [comment]
I suggest try another USB drive. What are you using currently? The current Goteks have very little RAM for buffering.

**Comment by Prince-Eugen** [comment]
voila....it works with another no name 16gb usb. I have used verbatim 16g 2.0 , but it seems that it had issue with gotek. 

issue solved, game save disk with dune 2 and sensible work flawlessly. thx for support.

**Reply by keirf** [reply]
No problem. Many usb drives have occasionally long delay when writing. With limited buffering on current goteks, data then gets lost.
